android/src/main/java/co/airbitz/fastcrypto/MoneroAsyncTask.java

```diff
@@ -63,22 +68,36 @@ protected Void doInBackground(Void... voids) {
                 connection.setDoOutput(true);
                 try (OutputStream outputStream = connection.getOutputStream()) {
                     for (int i = 0; i < requestLength; i++) {
+                        if (isStopped.get()) { 
+                            promise.reject("Err", new Exception("Operations are stopped"));
```

Comment:
Is it safe to use `promise.reject` instead of `promise.resolve` (like iOS) @mvayngrib @gutenye ? When I refreshed the monitor, no log was shown in the console log, but the app was exited.

Reply:
Why iOS always use resolve with err_msg for error case, but Android uses reject?

Reply:
Let me investigate with logcat.